### ISSUE TITLE
SMTP Issue (OpenSSL::SSL::SSLError)

### ISSUE THREAD

**Issue by valentin2105** [tootsuite/mastodon#1496]
Hi,

There is an issue with last commits,

I go hundred of Mail issue while sending them :

OpenSSL::SSL::SSLError: SSL_connect returned=1 errno=0 state=error: certificate verify failed

(sidekiq logs)

Any idea what it it ? 

I use MailGun, so I don't think they forget to renew they certificates.

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by mjankowski** [comment]
What's your general server setup and how is ruby installed?

**Comment by valentin2105** [comment]
I use Dockerfile and Docker-compose.yml from the Git Repo.
Not any specific change on my configuration.

( It worked from days)

**Comment by valentin2105** [comment]
It's exactly the same issue (I don't comment on this, because is not a Gandi issue. )

**Comment by ToroNZ** [comment]
Same for me with SparkPost:

OpenSSL::SSL::SSLError: SSL_connect returned=1 errno=0 state=error: certificate verify failed

**Comment by ToroNZ** [comment]
Can confirm that after following these changes it works.
[https://github.com/tootsuite/mastodon/pull/1486/files](url)

```
docker-compose stop
docker-compose build
docker-compose up -d
```

And it's running fine.

**Comment by kyzh** [comment]
@valentin2105 did you check the security rule/firewall rule?
They block smtp out by default.
@ToroNZ can you try using the change in #1486

**Comment by zaksoup** [comment]
I just started seeing this with Mailgun. Will check PR #1486

**Comment by ToroNZ** [comment]
Yeap, #1486 solves it.

**Comment by wxcafe** [comment]
@valentin2105 please confirm that your problem is fixed and close the issue

**Comment by valentin2105** [comment]
I have checked out to tag v1.1.1, so I don't know if it's fixed, it work in 1.1.1.
I think #1486  will fix it.
